carbonmark-api/src/models/User.model.ts

```diff
@@ -0,0 +1,20 @@
+import { Static, Type } from "@sinclair/typebox";
+import { Activity } from "./Activity.model";
+import { Listing } from "./Listing.model";
+import { Nullable } from "./Utility.model";
+
+export const User = Type.Object({
+  handle: Type.String(),
+  username: Type.String(),
+  description: Type.String(),
+  profileImgUrl: Nullable(Type.String()),
+  updatedAt: Type.Number(),
+  createdAt: Type.Number(),
+  wallet: Type.String(),
+  listings: Type.Array(Listing),
+  activities: Type.Array(Activity),
+  //@todo replace unknown with Holding model type
+  assets: Type.Array(Type.Unknown()),
```

Comment:
cant slap this in real quick? so we dont forget

here is a sample `asset` (aka Holding?) from the prod api
```json
    {
      "id": "0x20a580444dd4a90cc8990da7b480c5e3d605a26f1d9121d3a940e71929d4f6a7707b159a585cbb97",
      "token": {
        "id": "0x1d9121d3a940e71929d4f6a7707b159a585cbb97",
        "name": "C3 Tokens: C3T-VCS-51-2014",
        "symbol": "C3T-VCS-51-2014",
        "decimals": 18
      },
      "amount": "30000000000000000"
    }
 ```

Reply:
Yep done, see `Asset.model.ts`